src/CardListItem/CardListItem.js

```diff
@@ -2,49 +2,40 @@ import React, { Component } from "react";
 import "./CardListItem.css";
 
 class CardListItem extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      count: props.userCardsData.find(card => {
-        return card.cardName === props.cardName;
-      }).cardCount
```

Comment:
The CardListItem component no longer needs to hold state. It was unnecessary since the data it had was just a subset of the data in App's userCardsData.

Reply:
I looked it up and yes, the constructor is not necessary. "if you don’t initialize state and you don’t bind methods, you don’t need to implement a constructor for your React component."  Since we are using the arrow functions we are not binding the event handlers in the constructor and can get rid of them.  Nice work Jeo!